spec/support/form_helpers.rb

```diff
@@ -484,6 +476,11 @@ def localise_date_from_input_fields(year:, month:, day:)
     I18n.l(Date.parse("#{year}-#{month}-#{day}"))
   end
 
+  def financial_quarter_and_year_from_date_input_fields(year:, month:, day:)
+    date = Date.parse("#{year}-#{month}-#{day}")
```

Comment:
Would `Date.new(year, month, day)` be safer?

Reply:
They're strings, and `Date.new(year.to_i, month.to_i, day.to_i)` looked like an eyesore, so I went with what the localisation method did.